runtime/lua/vim/_extui/shared.lua

```diff
@@ -47,8 +47,8 @@ function M.tab_check_wins()
         local parser = assert(vim.treesitter.get_parser(M.bufs.cmd, 'vim', {}))
         M.cmd.highlighter = vim.treesitter.highlighter.new(parser)
       elseif type == 'more' then
-        -- Close more window with Ctrl-C.
-        vim.keymap.set('n', '<C-c>', '<C-w>c', { buffer = M.bufs.more })
+        -- Close more window with `q`, same as `checkhealth`
+        vim.keymap.set('n', 'q', '<C-w>c', { buffer = M.bufs.more })
```

Comment:
btw, if anyone says "this shouldn't override my `q` mapping", that doesn't matter. Because for most users, having to *restore* that mapping for the pager is a bigger cost than the inverse.

There needs to be some guarantteed single-key way to exit the pager. Overriding that should be opt-in, not opt-out.

Reply:
As far as I see it, the two main interactions for the message window are
1. searching,
2. yanking.

Any (reasonably common) mapping related to that is privileged and should not be clobbered. Everything else is up for grabs; this includes `q` for macro recording.